uber/site_sections/preregistration.py

```diff
@@ -16,9 +16,7 @@ def check_prereg_reqs(attendee):
 
 
 def check_dealer(group):
-    if not group.address:
-        return 'Dealers are required to provide an address for tax purposes'
-    elif not group.wares:
+    if not group.wares:
         return 'You must provide a detailed explanation of what you sell for us to evaluate your submission'
```

Comment:
This version of the address is deprecated - we now collect the attendee's address if they're a dealer, instead of a "group" address.